### ISSUE TITLE
Not really a bug, but a request to update the github project’s url.

### ISSUE THREAD

**Issue by ytrezq** [NigelCunningham/tuxonice-kernel-old#20]
At the top of https://github.com/NigelCunningham/tuxonice-kernel, the project’s website address is selected for  http://tuxonice.net/.

As tuxonice.net is down, please update this link !

**Comment by hjudt** [comment]
Yes, and please update the patchset for the latest kernel versions too! We need tuxonice, it is still much better than vanilla kernel hibernation! Please keep up the good work!

**Comment by ytrezq** [comment]
@hjudt : Yep, even the user space hibernation crash on resume in my case whatever the kernel is. Only tuxonice works for me.

The working alternative to Tuxonice is called Windows® and support Linux binaries.

**Comment by hmatrjp** [comment]
http://tuxonice.net/ seems to be used by the other company now.

I believe http://tuxonice.nigelcunningham.com.au/ is the right one for current developers and users.
And this site is down showing php error message.
Maybe the site master can't take time to change a setting while he needs set 'mysqli' instead of 'mysql' in the configuration to use drupal with php7.

Still, download archive is available: http://tuxonice.nigelcunningham.com.au/downloads/all/

**Comment by ytrezq** [comment]
@hmatrjp : @NigelCunningham doesn’t replies to my e‑mails. But maybe it’s because I took his public address which makes he couldn’t sort it among spams.

**Comment by hmatrjp** [comment]
He is in the years of study, related to theology if my memory is correct, thus may disconnected from internet, I guess. Few years ago, I have read it on that web site. Is this info too old ?

**Comment by ytrezq** [comment]
@hmatrjp : he contribute to one of his GitHub’s projects this month.

**Comment by NigelCunningham** [comment]
Sorry for my slow response, guys. I have a number of projects on the go, but will try to get TuxOnIce some attention soon.

I'm not studying - just very busy.

**Comment by joostheijkoop** [comment]
I would like to volunteer to do anything I can help with

**Comment by hjudt** [comment]
Hi Nigel, any news or updates on this? Your last reply has been already one month ago.

**Comment by hjudt** [comment]
Hi Nigel, my last try :-/ How about an update? Any chance for another round of tuxonice? Or is it really dead now?

**Comment by NigelCunningham** [comment]
Hi Harald.

I've gotten the repo back up to date and have been updating from there so TuxOnIce is not dead. I've also been working on a new website (the old one was Drupal 6), but only slowly. As always, too many projects on the go at once.

Is there something in particular you're looking for? (Patches rather than the repo?)

**Comment by hjudt** [comment]
Hi Nigel, glad to hear from you again. Thanks for bringing the repo back up to date. Cloning the repo is fine for me, I can create a patch using git if I need but usually I use a clone of the repo. I will test it soon, it's been nearly a year that hibernation and resuming worked reliably on my machines, only recently it was fixed it in vanilla and I hope that tuxonice will work again too. Looking forward to a maintained tuxonice and the new website.

**Comment by hjudt** [comment]
I have successfully updated to tuxonice for 4.10.1, so thanks a lot for the update. I will still have to test more cycles to see if it is really stable, but it seems to be so far.
I've found only one minor caveat: It doesn't work with Con Kolivas' new MuQSS scheduler (the successor to BFS, available [here](http://ck-hack.blogspot.co.at/2017/02/linux-410-ck1-muqss-version-0152-for.html)). There is always the message "compress read error -22" when resuming and the process hangs, while the UI still reacts to keys. It doesn't occur with the default CFS, though, and vanilla hibernation also works with MuQSS. Could there still be a bug in tuxonice?

**Comment by hjudt** [comment]
Unfortunately, it doesn't seem to work reliably even when applied on vanilla. It doesn't look like I can attach an image here, and last time I tried to send you a mail that didn't work. So let's try imgur: http://imgur.com/995GYC8
This happened when writing the image.

**Comment by hjudt** [comment]
I believe the last crash (see screenshot) was due to an ext4 extend problem which has been fixed in some 4.10.x. However, after several hibernate/resume cycles tuxonice froze during resume so there still has to be some other error. Vanilla hibernation works fine.
